pinc/dpsql.inc

```diff
@@ -2,22 +2,20 @@
 define("DPSQL_NO_RANK", false);
 define("DPSQL_SHOW_RANK", true);
 
-define("DPSQL_NO_RIGHT_ALIGN_INTS", false);
-define("DPSQL_RIGHT_ALIGN_INTS", true);
-
-function dpsql_dump_query($query, $start_at = 0, $show_rank = DPSQL_NO_RANK)
+function dpsql_dump_query(string $query, int $start_at = 0, bool $show_rank = DPSQL_NO_RANK): void
 {
     $result = DPDatabase::query($query);
     dpsql_dump_query_result($result, $start_at, $show_rank);
 }
 
-function dpsql_dump_themed_query($query, $start_at = 0, $show_rank = DPSQL_NO_RANK, $ralign_ints = DPSQL_RIGHT_ALIGN_INTS)
+function dpsql_dump_themed_query(string $query, int $start_at = 0, bool $show_rank = DPSQL_NO_RANK): void
 {
     $result = DPDatabase::query($query);
-    dpsql_dump_themed_query_result($result, $start_at, $show_rank, $ralign_ints);
+    dpsql_dump_themed_query_result($result, $start_at, $show_rank);
 }
 
-function dpsql_dump_query_result($result, $start_at = 0, $show_rank = DPSQL_NO_RANK)
+/** @param bool|mysqli_result $result */
+function dpsql_dump_query_result($result, int $start_at = 0, bool $show_rank = DPSQL_NO_RANK): void
 {
     $n_cols = mysqli_num_fields($result);
```

Comment:
While the input type is correct as it's the return value from `DPDatabase::query`, I don't think this line will work with a `bool`.